airflow/jobs.py

```diff
@@ -272,13 +272,33 @@ def manage_slas(self, dag, session=None):
             .filter(SlaMiss.dag_id == dag.dag_id)
             .all()
         )
+
+        current_run_date = (datetime.now() -
+                            dag.schedule_interval).isoformat()[:10]
```

Comment:
This seems arbitrary, wouldn't work with non-daily DAG, can we just work off of the list we're about to send an email about?